src/schema/schema_drop.c

```diff
@@ -223,11 +251,11 @@ __drop_tiered(WT_SESSION_IMPL *session, const char *uri, bool force, const char
             WT_PREFIX_SKIP_REQUIRED(session, filename, "file:");
             WT_ERR(__wt_meta_track_drop(session, filename));
         }
-        tiered->tiers[WT_TIERED_INDEX_LOCAL].tier = NULL;
+        tiered_tmp.tiers[WT_TIERED_INDEX_LOCAL].tier = NULL;
```

Comment:
I don't think NULL-ing out this entry has any affect, since `tiered_tmp` is a stack variable, and this is not referred to later in this function.